CMakeLists.txt

```diff
@@ -209,6 +209,17 @@ set_property(TARGET ${PROJECT_NAME} PROPERTY POSITION_INDEPENDENT_CODE ON)
 # INSTALL
 #################################################################################
 
+# libmd.pc for pkg-config
+if (CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
+   set(DEST_DIR /usr/local)
+else()
+    set(DEST_DIR ${CMAKE_INSTALL_PREFIX})
```

Comment:
[`/usr/local` is already the default prefix](https://cmake.org/cmake/help/latest/variable/CMAKE_INSTALL_PREFIX.html#variable:CMAKE_INSTALL_PREFIX) , so the block above is equivalent to `set(DEST_DIR ${CMAKE_INSTALL_PREFIX})`.